<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>965</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>781</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="AccountTable">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>80</y>
      <width>781</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTableView" name="AmountTable">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="search_account">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_amount">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>40</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QComboBox" name="stock_codes">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>40</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_id">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>135</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Account Id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_nm">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>135</width>
      <height>27</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Korean" country="SouthKorea"/>
    </property>
    <property name="placeholderText">
     <string>Account Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="full_account_id">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>135</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Account Id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_nm_eng">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>135</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Account name eng</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Account_id">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>135</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Account Id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="account_nm_kor">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>540</y>
      <width>135</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="locale">
     <locale language="Korean" country="SouthKorea"/>
    </property>
    <property name="placeholderText">
     <string>Account name kor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>600</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Insert">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>600</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>671</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="ConfirmAccountTable"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Update">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>540</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Backup">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>590</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Backup</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1680</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
